### ISSUE TITLE
Hide toolbars on scroll

### ISSUE THREAD

**Issue by CTalvio** [liftoff-app/liftoff#43]
Basically this:

https://github.com/liftoff-app/liftoff/assets/4365015/17755dc3-e6ed-4078-b6b1-b886e6d61ee0

**Comment by CTalvio** [comment]
#55 Only works for the App Bar? Not the the home tab?

**Comment by zachatrocity** [comment]
oh the bottom nav bar should disappear too. Didn't see that in your video cause the controls. Thanks for checking back!

**Comment by CTalvio** [comment]
Yes, its a very handy context sensitive little QoL improvement. It's near universal in web browsers, and as such, very intuitive to users as well.

Looking forwards to it affecting the second UI bar as well :D

**Comment by zachatrocity** [comment]
We'll get to it eventually. Lower priority as we focus on bugs

**Comment by ZeusOfTheCrows** [comment]
i know creating options is annoying, but i think this would be good to have as an optional feature - i personally always disable this on apps where i can do so
